code/_global_vars/lists/flavor.dm

```diff
@@ -7,4 +7,45 @@ GLOBAL_LIST_INIT(phonetic_alphabet, list("Alpha", "Bravo", "Charlie",
 	"Delta", "Echo", "Foxtrot", "Golf", "Hotel", "India", "Juliet",
 	"Kilo", "Lima", "Mike", "November", "Oscar", "Papa", "Quebec",
 	"Romeo", "Sierra", "Tango", "Uniform", "Victor", "Whiskey", "X-ray",
-	"Yankee", "Zulu"))
\ No newline at end of file
+	"Yankee", "Zulu"))
+
+GLOBAL_LIST_INIT(music_tracks, list(
+	"Beyond" = /music_track/ambispace,
```

Comment:
Do you have any idea about the copyright status of these tracks?

Reply:
ahh i see licensing information is included below.

We can't have any tracks which disallow commercial use, it will be necessary to look through the licenses and possibly remove some of these